index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<meta http-equiv="X-UA-Compatible" content="IE=edge">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<link rel="stylesheet" href="style.css">
+	<title>Portfolio</title>
+</head>
+<body>
+	<section class="screen_container">
+		<header>
+			<nav>
+				<section class="toolbar_container">
+					<p class="Title_logo">My Logo</p>
+					<img src="./images/i.png"  class="logo" alt="Icon menu">
+				</section>
+			</nav>
+		</header>
+		
+		<main class="main_container">
+			<section class="text_container">
+				<p class="paragraphe">Hey There. <br/>I’m Leader.</p>
+				
+				<h4>I'm a Software Developer</h4>
+				<p class="para">I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+					don’t hesitate to contact me.</p>
+					<img  class="ilustration" src="./images/Illustration.png" alt="images">
+			</section>
+			
+			<section class="icons-container">
+				<div class="icons-containers">
+					<img src="./images/Normal Button/Tertiary/Icons/Icon -GitHub.png"/>
+					<img src="./images/Icon - Linkedin.png"/>
+					<img src="./images/Normal Button/Tertiary/Enabled.png"/>
+					<img src="./images/Normal Button/Tertiary/Icons/Icon - Twitter.png"/>
+					<img src="./images/Normal Button/Tertiary/Icons/Icon - Medium.png"/>
+				</div>				
```

Comment:
- [ ] Please use a `ul` tag to group the media icons, `li` tag to wrap each icon, and each icon has a link `a` tag. Apart from being semantically correct, the `a` tag will let you connect your social networks with the icons, using `href` attribute.